app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -114,6 +126,8 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
                 }
             }
         })
+
+        Toast.makeText(context, R.string.remove_tile_toast_title, Toast.LENGTH_LONG).show()
```

Comment:
This is fine but just thoughts about the UX:
- Tif told me the spec was it shows up when we click tiles (but that seems wrong since it's too late)
- This also seems annoying to show every time, but better than ^

Let's land and see what people think.